service/src/main.rs

```diff
@@ -782,3 +789,33 @@ fn bootstrap_funds_from_alice(
 	api.signer = signer_orig;
 	Ok(())
 }
+
+/// In case this is the first validateer on the specified parentchain, we need to trigger
+/// the parentchain block import up until the block where we have registered ourselves.
+fn bootstrap_parentchain_blocks_for_primary_validateer<E: EnclaveBase + TeerexApi + Sidechain>(
+	node_api: &Api<sr25519::Pair, WsRpcClient>,
+	enclave_api: Arc<E>,
+	last_synced_header: &Header,
+	registry_block_hash: Option<Hash>,
```

Comment:
`registry` is block hash is too ambiguous in my opinion. I'd prefer `register_enclave_xt_hash`.

Reply:
Agreed, thanks for your proposal